Sources/TuistKit/GraphMappers/GraphMapperProvider.swift

```diff
@@ -27,6 +28,8 @@ final class GraphMapperProvider: GraphMapperProviding {
         if useCache {
             mappers.append(CacheMapper())
         }
+        
+        mappers.append(SigningMapper())
```

Comment:
Awesome!

Reply:
What about adding a test to make sure that the mapper is provided by the `GraphMapperProvider`?